Jenkinsfile

```diff
@@ -202,27 +203,30 @@ def get_macos_pipeline()
 {
     return {
         stage("macOS") {
-            node ("macos") {
-            // Delete workspace when build is done
-                cleanWs()
+           timestamps {
```

Comment:
I do not understand the usefulness of `timestamps` here. Especially considering that Jenkins is pretty good about timing stages without it.

Reply:
well we don't learn much about why things are slow from jenkins, only the build stages.